deliver/lib/deliver/app_screenshot.rb

```diff
@@ -137,7 +137,7 @@ def self.devices
     def self.calculate_screen_size(path)
       size = FastImage.size(path)
 
-      UI.user_error!("Could not find or parse file at path '#{path}'") if size.nil? or size.count == 0
+      UI.user_error!("Could not find or parse file at path '#{path}'") if size.nil? or size.count.zero?
```

Comment:
These are actually not changes we want in here. We recently updated the rubocop rules to not include this